### ISSUE TITLE
Switch to (fragments of) Python grammar from PyGL

### ISSUE THREAD

**Issue by gvanrossum** [gvanrossum/pegen#2]
It would be good if pegen switched from using fragments of the sketch of the PEG Python grammar in the TatSu examples (which came from an untested ANTLR grammar), and switched to fragments from the one in [PyGL](https://github.com/neogeny/pygl) (which I've been testing, and came from Grammar/Grammar, which is authoritative).

@apalala (Since I'm quoting you here. :-)

**Comment by apalala** [comment]
Which is the grammar with which you got 100% coverage of `cpython/**/*.py` @gvanrossum ?

We should work with PyGL's grammar because of Grammar/Grammar, and because I had already done the refactoring work to make it PEG, and to parse `cpython/**/*.py`. There's no efficient way to assess the quality of the grammar from ANTLR/grammars, and that grammar risks being too ANTLR-lile.

**Comment by gvanrossum** [comment]
The grammar that got me 100% coverage is `data/simpy.gram` in this project. I did not start from ANTLR. I looked closely at Grammar/Grammar but did not follow it exactly in all cases (because I want to showcase pegen's ability to have a grammar that looks more natural than the old pgen-based grammar).

I'd be happy to compare it with PyGL's grammar -- it should be simple to write a translator from PyGL to pegen's format, or alternatively we could create a new [metagrammar](https://medium.com/@gvanrossum_83706/a-meta-grammar-for-peg-parsers-3d3d502ea332?source=friends_link&sk=564d233354234081eae1d0b21ad7592c) that can parse PyGL directly.
